packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -832,6 +833,13 @@ export const useAzureCommunicationCallWithChatAdapter = (
   return adapter;
 };
 
+/* @conditional-compile-remove(teams-inline-images) */
+/**
+ * Configuration options to include when creating CallWithChatAdapter.
+ * @beta
+ */
+export type CallWithChatOptions = ChatAdapterOptions;
```

Comment:
Again, if that is just the same type, we can just directly use ChatAdapterOptions, without introducing a new type here for now

Reply:
Sry, AzureCommunicationChatAdapterOptions, to be more accurate